spec/search_spec.rb

```diff
@@ -71,6 +71,24 @@ module Search
                     QueryGraph::Graph.from_path([user.id_field]))
         ]
       end
+
+      it 'total cost increases when new update is added' do
```

Comment:
This might be better written as

`it 'increases the total cost when an update is added'`

The goal is that it should read as something that the search code does with "it" being part of the sentence. The test looks great though!